Workbooks/Azure Backup/ASR ReplicatedItems /ASR ReplicatedItems.workbook

```diff
@@ -0,0 +1,1220 @@
+            "name": "Subscriptions",
+            "type": 6,
+            "description": "Subscriptions to filter the list of workspaces",
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "value": [
+              "value::all"
+            ],
+            "typeSettings": {
+              "additionalResourceOptions": [
+                "value::all"
+              ],
+              "includeAll": false,
+              "selectAllValue": ""
+            },
+            "label": "Workspace Subscription"
+          },
+          {
+            "id": "2373a24f-ad32-4909-a7f6-59b373dcde6c",
+            "version": "KqlParameterItem/1.0",
+            "name": "Workspaces",
+            "label": "Workspace",
+            "type": 5,
+            "description": "LA Workspaces configured in vault diagnostic settings",
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "where type =~ 'microsoft.operationalinsights/workspaces' | project id",
+            "crossComponentResources": [
+              "{Subscriptions}"
+            ],
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "queryType": 1,
+            "resourceType": "microsoft.resourcegraph/resources",
+            "value": []
+          },
+          {
+            "id": "101b128c-36de-4a8f-a0c9-27548aa083ed",
+            "version": "KqlParameterItem/1.0",
+            "name": "ReplicationScenario",
+            "label": "Replication Scenario",
+            "type": 2,
+            "description": "Filter for replication scenario",
+            "isRequired": true,
+            "typeSettings": {
+              "additionalResourceOptions": []
+            },
+            "jsonData": "[    \r\n{ \"value\": \"Azure\", \t\t\t\t\t\t\"label\": \"Azure\" },\r\n{ \"value\": \"Hybrid\", \t\t\t\"label\": \"Hybrid\" }\r\n]",
+            "value": "Hybrid"
+          }
+        ],
+        "style": "above",
+        "doNotRunWhenHidden": true,
+        "queryType": 1,
+        "resourceType": "microsoft.resourcegraph/resources"
+      },
+      "customWidth": "100",
+      "name": "Filters1"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "_____________________"
+      },
+      "name": "Line1"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "## Replication Health History"
+      },
+      "conditionalVisibility": {
+        "parameterName": "Workspaces",
+        "comparison": "isNotEqualTo"
+      },
+      "name": "Heading2"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "This report helps you view information on your replicated items over a specified duration of time. [Learn more](https://go.microsoft.com/fwlink/?linkid=2211463&clcid=0x409)"
```

Comment:
Please change to - This report provides information on the replicated items over a specified duration of time.